docs/apps/srt-test-relay.md

```diff
@@ -0,0 +1,34 @@
+## srt-test-relay
```

Comment:
Again, the level of heading is incorrect. Please change ## to #.

Reply:
I've also noticed that in srt-tunnel document the headings formatting is different. Could you please update it to look similar? Plus please change the title from "SRT Tunnel" to "srt-tunnel" for consistency. Thanks!